kenware/Assessment
# #34747487 add more functionality in question endpoints

#### What does this PR do?
- add more functionality to the question endpoints

#### Description
- increase assessment total mark when a question is added
- reduce the assessment total mark when a question is deleted

#### How should this be manually tested?
- Create a local copy `git fetch ft-add-more-functionality-in-question-endpoints-34747487`
- Checkout to the branch `git checkout ft-add-more-functionality-in-question-endpoints-34747487`
- Start the application with the command: `python manage.py runserver`
- On postman make a GET or POST request to `{{base-url}}/api/v1/questions/`
- On postman make a PATCH or DELETE request to `{{base-url}}/api/v1/questions/<question_id>/`
- An authenticated user should be able to carry out all crude operation on user endpoints
- To run tests use `python manage.py`
#### Any background context you want to provide?
- N/A

#### What are the relevant pivotal tracker stories?
- [#162187176](https://www.pivotaltracker.com/story/show/162187176)
